ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/commons/objectivefunction/InitialSensitivityAnalysisObjectiveFunctionCreator.java

```diff
@@ -34,7 +35,7 @@ protected List<CostEvaluator> getVirtualCostEvaluators(MarginEvaluator marginEva
         List<CostEvaluator> virtualCostEvaluators = new ArrayList<>();
 
         if (raoParameters.getObjectiveFunctionParameters().getType().costOptimization()) {
-            virtualCostEvaluators.add(new MinMarginViolationEvaluator(flowCnecs, unit, marginEvaluator));
+            virtualCostEvaluators.add(new MinMarginViolationEvaluator(flowCnecs, unit, marginEvaluator, raoParameters.getExtension(OpenRaoSearchTreeParameters.class).getMinMarginsParameters().orElseThrow().getShiftedViolationPenalty()));
```

Comment:
could you rename overloadPenalty attribute in MinMarginViolationEvaluator to be consistent with renaming please